lib/commands/-command.js

```diff
@@ -6,7 +6,7 @@ module.exports = Command.extend({
   init() {
     process.env.EMBER_CLI_ELECTRON = true;
 
-    if (this._super && this._super.init) {
+    if (this._super && this._super.init && this._super.init !== this.init) {
```

Comment:
Total conjecture: could my issues have stemmed from using an older e-cli version; and @kevingelion's from keeping the `.apply(...)` syntax?